### PR TITLE
🐛 Handle the scoring of ignore policies

### PR DESCRIPTION
**Pull request by czunker**
This fixes the handling of ignored policies.
The policy controls are scored, but the score isn't added to the asset

### CODE REVIEW

File: policy/resolver.go [hunk 1]
```diff
@@ -404,14 +404,16 @@ func (s *LocalServices) tryResolve(ctx context.Context, policyMrn string, assetF
 		return nil, err
 	}
 
-	var rp *ResolvedPolicy
-	rp, err = s.DataLake.CachedResolvedPolicy(ctx, policyMrn, allFiltersChecksum, V2Code)
+	// var rp *ResolvedPolicy
+	_, err = s.DataLake.CachedResolvedPolicy(ctx, policyMrn, allFiltersChecksum, V2Code)
```
Comment (czunker) [inline policy/resolver.go]: The caching does currnetly not work. This is WIP and needs a fix.

File: policy/policy.go [hunk 2]
```diff
@@ -433,6 +433,7 @@ func (p *Policy) updateAllChecksums(ctx context.Context,
 			}
 
 			executionChecksum = executionChecksum.Add(ref.Mrn)
+			executionChecksum = executionChecksum.Add(ref.Action.String())
```
Comment (arlimus) [inline policy/policy.go]: Go for `AddUint` here, faster and smaller to do

Comment (czunker) [reply policy/policy.go]: Fixed.

File: policy/resolver.go (hunk 1 above)
Comment (czunker) [reply policy/resolver.go]: Fixed

**Review by arlimus — APPROVED** [review]
lgtm, thank you!

**Review by arlimus — APPROVED** [review]
Thanks for the update!

File: policy/policy.go [hunk 3]
```diff
@@ -433,6 +433,7 @@ func (p *Policy) updateAllChecksums(ctx context.Context,
 			}
 
 			executionChecksum = executionChecksum.Add(ref.Mrn)
+			executionChecksum = executionChecksum.AddUint(uint64(ref.Action))
```
Comment (joelddiaz) [inline policy/policy.go:436]: Does the line right above this have any purpose if the result in `executionChecksum` is immediately overwritten?

Comment (czunker) [reply policy/policy.go:436]: It's not overwritten, it get's additional data added:
https://github.com/mondoohq/cnquery/blob/323c116a72e37534ebeafc2ccfe9c6bc71f79e7b/checksums/checksum.go#L22

Another example for this in this file:
https://github.com/mondoohq/cnspec/blob/e730ccf05254e433192557aa73eb98effc123688/policy/policy.go#L366-L376

Comment (joelddiaz) [reply policy/policy.go:436]: Wow...I completely misread these two lines of. Just pretend I never said anything :)

**Pull request merged**